client/src/app/app-routing.module.ts

```diff
@@ -4,6 +4,7 @@ import { HomeComponent } from './components/home/home.component';
 import { AuthGuard } from './auth.guard';
 import { LoginComponent } from './components/login/login.component';
 import { ProfileComponent } from './components/profile/profile.component';
+import { KeithTestComponent } from './components/keith-test/keith-test/keith-test.component';
```

Comment:
Just to keep the commit history fairly clean I would prefer to name the component what we plan to have it in the future - i.e. `daily-log-form` or similar, the content could stay the same but I think this would be best just to keep things consistent.